app/pages/Help.js

```diff
@@ -19,56 +20,361 @@ class Help extends React.Component {
+                      <li>Start SWEX as administrator</li>
+                      <li>Make sure the profile exporter is enabled in swex settings</li>
+                    </ul>
+                  </li>
+                  <li>
+                    I can't play the game with HTTPS mode disabled:
+                    <ul>
+                      <li>Are you connected to a network? → Does that network have access to the internet?</li>
+                    </ul>
+                  </li>
+                </ul>
+              </li>
+            </ol>
+          </AccordionContent>
+
+          {/* Common issues in setting up SWEX */}
+          <Accordion.Title active={activeIndex === 7} index={7} onClick={this.handleAccordionClick.bind(this)}>
+            <Icon name="dropdown" />
+            Common issues in setting up SWEX
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 7}>
+            <p>
+              <ol>
+                <li>
+                  Firewall & VPN
+                  <ul>
+                    <li>
+                      Check if SWEX is allow by your default firewall{' '}
+                      <a href="https://drive.google.com/file/d/1o3ar804O27j3E_VyDpn4MtAl6R_JjeY-/view?usp=sharing" target="_blank">
+                        [Link to PDF Guide (windows 10)]{' '}
+                      </a>
+                      . If you can't find SWEX in the firewall, Add another app → add SWEX.
+                    </li>
+                    <li>If you use extra firewall (like McAfee) disable it</li>
+                    <li>If you have extra firewall on your phone disable it</li>
+                  </ul>
+                </li>
+                <li>
+                  WiFi
+                  <ul>
+                    <li>If you use any VPN disable it</li>
+                    <li>Check if you are on same wi-fi (not public)</li>
+                    <li>Check your phone / emulator wifi Settings, if you saved proxy settings</li>
+                  </ul>
+                </li>
+                <li>
+                  Other
+                  <ul>
+                    <li>Try to run SWEX as administrator</li>
+                    <li>
+                      Clear cert folder (you need to clear 2 folder, both file path and setting path){' '}
+                      <a href="https://drive.google.com/file/d/1B5mA-wbDWhyCjJvWOh4fTzEAcC9CKa96/view" target="_blank">
+                        {' '}
+                        [Windows Manual]
+                      </a>
+                    </li>
+                  </ul>
+                </li>
+              </ol>
+            </p>
+          </Accordion.Content>
+
+          {/* Errors by OS */}
+          <Accordion.Title active={activeIndex === 8} index={8} onClick={this.handleAccordionClick.bind(this)}>
+            <Icon name="dropdown" />
+            Errors by OS
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 8}>
+            <p>
```

Comment:
Its usually not great practise to wrap a list in a p element. I suggest to just remove the <p\> element and use solely the <ol\>